src/client/app.js

```diff
@@ -1,5 +1,5 @@
 var module = angular.module('app', ['cla.filters',
-    'ui.utils',
+    // 'ui.utils',
```

Comment:
Is this on purpose? If it is, we can remove the line

Reply:
ops